modules/navier_stokes/doc/content/source/fvbcs/INSFVAveragePressureValueBC.md

```diff
@@ -0,0 +1,14 @@
+# INSFVAveragePressureValueBC
+
+This object wraps [FVBoundaryIntegralValueConstraint.md], meaning that this
+object can be used to impose an average pressure value on a boundary. The
+`variable` parameter should correspond to the pressure variable. When applying a
+`INSFVAveragePressureValueBC` for the pressure, no `FVBCs` should be given for
+the velocity on the same `boundary`. In this way a zero viscous flux is
```

Comment:
> When applying a `INSFVAveragePressureValueBC` for the pressure, no `FVBCs` should be given for
the velocity on the same `boundary`.

Is there something that makes this conceptually impossible? That is actually what I am trying to do in the 
approach that we discussed earlier.

Reply:
When no FVBCs are applied for the velocity, then the momentum advection flux kernel runs on the boundary using data extrapolated from the interior. For the Euler equations (which obviously these are not), this type of boundary treatment is required: the eigenvalues for sub-Mach flow dictate that you have n-dim + 1 physical boundary conditions and one implicit boundary condition at an inflow boundary and one physical boundary condition and n-dim + 1 implicit boundary conditions at an outflow boundary.

But setting that aside... from the meeting you want to impose a `WCNSFVMomentumFluxBC` on the same boundary as this new `INSFVAveragePressureValueBC`? If that is the case, then yes, this is actually possible and will work with the code we have now. I can just amend this documentation